### PR TITLE
New Resource: `azurerm_kubernetes_fleet_manager`

### PR DESCRIPTION
**Pull request by hc-github-team-tf-azure**
Regenerating the Terraform Provider based on the latest changes

### CODE REVIEW

File: internal/services/containers/kubernetes_fleet_manager_resource_gen.go [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+					},
+					"kubernetes_version": {
+						Computed: true,
+						Type:     pluginsdk.TypeString,
+					},
+				},
+			},
+			ForceNew: true,
+			MaxItems: 1,
+			Optional: true,
```
Comment (tombuildsstuff) [inline internal/services/containers/kubernetes_fleet_manager_resource_gen.go:72]: this should probably be marked as required

File: website/docs/r/kubernetes_fleet_manager.html.markdown [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+The `hub_profile` block supports the following arguments:
+
+* `dns_prefix` - (Required) 
+
+The `hub_profile` block exports the following arguments:
+
+* `fqdn` - 
+
+* `kubernetes_version` -
```
Comment (tombuildsstuff) [inline website/docs/r/kubernetes_fleet_manager.html.markdown:64]: these need mapping through

File: internal/services/containers/registration_gen.go [hunk 3]
```diff
@@ -1,26 +1,30 @@
...
+	return []sdk.Resource{
+		KubernetesFleetManagerResource{},
+	}
 }
 
-func (a autoRegistration) WebsiteCategories() []string {
-	return []string{}
+func (autoRegistration) WebsiteCategories() []string {
+	return []string{
+		"Kubernetes Fleet Manager",
```
Comment (tombuildsstuff) [inline internal/services/containers/registration_gen.go:28]: this should be `Containers`

**Review by tombuildsstuff — APPROVED** [review]
A few comments inline to fix in the next regeneration but this otherwise seems fine

**Review by katbyte — APPROVED** [review]
LGTM ☂️

**Pull request merged**